frontend/src/scss/PrimeStyles.scss

```diff
@@ -189,8 +189,8 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
   max-height:
     calc(
       100vh - #{units(6)} - #{$results-dropdown-spacing}
-    ); // uses header height on small screens
-  overflow-y: scroll;
+    ); // uses header height on small screens// uses header height on small screens
+  overflow-y: auto;
```

Comment:
@maya I feel like the scroll here was a behavior you intentionally delivered in a PR not too long ago

Reply:
Checked this works, LGTM.